imap_processing/ultra/packet_definitions/P_U45_AUXILIARY.xml

```diff
@@ -358,14 +358,14 @@
 		</xtce:ParameterSet>
 		<!-- End metadata -->
 		<xtce:ContainerSet>
-			<xtce:SequenceContainer name="CCSDSPacket" abstract="true">
```

Comment:
@laspsandoval Do you know what `abstract="true"` does here? I had to remove it in order to get these tests to pass, and the other instrument's packet definitions don't have it. I am thinking that it is not important?

Reply:
I now see that removing this caused `test_decom_apid_88*.py` to fail, so I guess it is important afterall!